dsmr/block.go

```diff
@@ -111,38 +111,60 @@ func parseChunkProto[T Tx](chunkProto *dsmr.Chunk) (Chunk[T], error) {
 type ChunkFault struct{}
 
 type Block struct {
-	ID        ids.ID `serialize:"true"`
 	ParentID  ids.ID `serialize:"true"`
 	Height    uint64 `serialize:"true"`
 	Timestamp int64  `serialize:"true"`
 
-	// TODO Add TxsRoot?
-	// TODO Add OutgoingWarpRoot?
-	// TODO Do we need separate Available + Executed Chunks?
 	Chunks []*ChunkCertificate `serialize:"true"`
+
+	blkID    ids.ID
+	blkBytes []byte
```

Comment:
These are left unpopulated for now. It seems this type was not fully created, so leaving a correct block constructor for a follow up